web-wallet/src/style/dusk/language.css

```diff
@@ -106,7 +106,7 @@
 
   /* Ribbon */
 
-  --ribbon-color: var(--magnolia);
+  --ribbon-color: var(--smokey-black);
```

Comment:
As you are here fixing this, please get rid of these two variables that have no place in the language file.
Just use `--secondary-color` and `--on-secondary-color` in the main css file where the ribbon is styled.

Reply:
Agreed. Done!